data/index.js.map

```diff
@@ -1 +0,0 @@
-{"version":3,"file":"index.js","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":";;;IAEA,IAAM,CAAC,GAAG,IAAI,sBAAe,CAAC,KAAK,CAAC,CAAC"}
```

Comment:
Prob safe to drop this file?

Reply:
I removed this file and also removed data folder